src/main/java/TaskManager.java

```diff
@@ -0,0 +1,99 @@
+public class TaskManager {
+    private Task[] tasks = new Task[100];
+    private int numTask = 0;
+
+    public static final String BORDER = "_________________________________________\n";
```

Comment:
I like how descriptive your constants are! Makes it easier to follow your code in certain areas.